test/tests/ver-1jb/ver-1jb.i

```diff
@@ -0,0 +1,369 @@
+# Verification Problem #1jb from TMAP7 V&V document
+# Radioactive Decay of Tritium in a Distributed Trap
+
+# Physical Constants
+ideal_gas_constant = ${units 8.31446261815324 J/K/mol} # from PhysicalConstants.h
+boltzmann_constant = ${units 1.380649e-23 J/K } # from PhysicalConstants.h
+eV_to_J = 1.602176634e-19 # J/eV - from PhysicalConstants.h
+boltzmann_constant_eV = ${fparse boltzmann_constant / eV_to_J} # eV/K
+
+# Case and model parameters (adapted from TMAP7)
+slab_length = 1.5 # m
+tritium_mobile_concentration_initial = ${units 1 atoms/m3}
+trapping_sites_atomic_fraction_max = ${units 0.001 at.frac.}
+trapping_sites_fraction_occupied_initial = 0.5 # (-)
+normal_center_position = ${fparse slab_length/2} # m
+normal_standard_deviation = ${fparse slab_length/4} # m
+density_material = ${units 6.34e28 atoms/m^3} # for tungsten
+density_scalar = ${units 1e25 atoms/m^3} # used to scale variables and use a dimensionless system
+temperature = ${units 300 K} # assumed (TMAP7's input file lists 273 K)
+tritium_diffusivity_prefactor = ${units 1.58e-4 m^2/s} # from TMAP7 V&V input file
+tritium_diffusivity_energy = ${units 308000.0 J/K} # from TMAP7 V&V input file
+tritium_release_prefactor = ${units 1.0e13 1/s} # from TMAP7 V&V input file
+tritium_release_energy = ${units 4.2 eV}
+tritium_trapping_prefactor = ${units 2.096e15 1/s} # from TMAP7 V&V input file
+tritium_trapping_energy = ${tritium_diffusivity_energy} # J/K - from TMAP7 V&V input file
```

Comment:
I prefer to have it as a separate variable because it could be interpreted as a typo if it wasn't. It's quite common to assume that the diffusion energy is the same as the trapping energy, but it doesn't fully make sense to me. So I prefer to make it clear that this is intensional.

Reply:
Resolved but I can't make it official yet haha